test/webhook.js

```diff
@@ -423,6 +423,58 @@ describe('Webhooks', function() {
 		expect(event.target_user.id).to.eq(thierryID);
 	});
 
+	it('moderation ban', async function() {
+		const [events] = await Promise.all([
+			promises.waitForEvents('user.banned'),
+			client.banUser(tommasoID, {
+				user_id: jaapID,
+				reason: 'reason',
+				timeout: 123,
+			}),
+		]);
+		const event = events[0];
+		expect(event.reason).to.equal('reason');
+		expect(event.user.id).to.equal(tommasoID);
+		expect(event.expiration).to.exist;
+	});
+
+	it('moderation unban', async function() {
+		const [events] = await Promise.all([
+			promises.waitForEvents('user.unbanned'),
+			client.unbanUser(tommasoID, {
+				user_id: jaapID,
+			}),
+		]);
+		const event = events[0];
+		expect(event.user.id).to.equal(tommasoID);
+	});
+
+	it('slash ban', async function() {
+		const text = `/ban ${tommasoID} reason`;
+		const [events] = await Promise.all([
+			promises.waitForEvents('user.banned'),
+			chan.sendMessage({ text, user_id: jaapID }),
+		]);
+		const event = events[0];
+		expect(event.reason).to.equal('reason');
+		expect(event.channel_id).to.equal(chan.id);
+		expect(event.channel_type).to.equal(chan.type);
+		expect(event.expiration).to.exist;
```

Comment:
you should verify that we return a datetime object that we can parse the same way as we do for created_at

Reply:
I have not found any datetime checks. The only I've found is:
`expect(userResponse.created_at).to.not.equal('0001-01-01T00:00:00Z')`

created_at is a string, not an object.

```
{
  type: 'user.banned',
  created_at: '2019-11-05T12:13:52.092106Z',
  reason: 'reason',
  expiration: '2019-11-05T14:16:52.091473Z',
  received_at: 2019-11-05T12:13:52.092Z
}
```